scripts/pack.py

```diff
@@ -40,7 +39,7 @@ def pack(args):
 
     with ZipFile(str(args.output), 'w', ZIP_DEFLATED, **zkwargs) as zf:
         for path in sorted(args.directory.glob('**/*')):
-            if path.name[0] == '.' or path.name == 'meson.build':
+            if (path.name[0] == '.' or (args.exclude is not None and next((True for x in args.exclude if path.match(x)), False))):
```

Comment:
Well that's a mouthful. I'd advice defaulting `args.exclude` to `[]` (`default=[]` at `parser.add_argument` should do it), and using something like this for the condition: `any(path.match(x) for x in args.exclude)`